component/api.jsonnet

```diff
@@ -132,6 +132,10 @@ local objects = [
                       name: 'DEFAULT_API_SECRET_REF_NAME',
                       value: params.api.default_githost,
                     },
+                    {
+                      name: 'LIEUTENANT_INSTANCE',
+                      value: params.api.lieutenant_instance,
+                    },
```

Comment:
This should be done above (with the same approach as used for `STEWARD_IMAGE`), as we otherwise have two entries for env var `LIEUTENANT_INSTANCE` in the `env` array.

Reply:
Ah I see. Fixed it.